models/rtctrl_ctx_p.go

```diff
@@ -0,0 +1,99 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+)
+
+const (
+	DnrtctrlCtxP        = "uni/tn-%s/prof-%s/ctx-%s"
+	RnrtctrlCtxP        = "ctx-%s"
+	ParentDnrtctrlCtxP  = "uni/tn-%s/prof-%s"
```

Comment:
How does this works with the second DN: uni/tn-{name}/out-{name}/prof-{name}/ctx-{name} ?